### PR TITLE
changes

### PR DESCRIPTION
**Pull request by huaruiw**

### CODE REVIEW

File: src/game.js [hunk 1]
```diff
@@ -187,3 +187,6 @@ module.exports.registerControls = function() {
 		}
```
Comment (huaruiwu) [inline src/game.js]: looks good

File: src/pip.js [hunk 2]
```diff
@@ -29,3 +29,11 @@ class Pip {
 }
 
 module.exports = Pip
+
+// change 1
+//
+//
+//
+// change 3
```
Comment (huaruiwu) [inline src/pip.js]: looks good

File: src/pip.js [hunk 3]
```diff
@@ -29,3 +29,11 @@ class Pip {
 }
 
 module.exports = Pip
+
+// change 1
+//
+//
+//
```
Comment (huaruiwu) [inline src/pip.js]: looks good

File: src/pip.js [hunk 4]
```diff
@@ -29,3 +29,11 @@ class Pip {
 }
 
 module.exports = Pip
+
```
Comment (huaruiwu) [inline src/pip.js]: looks good

File: src/pip.js [hunk 5]
```diff
@@ -29,3 +29,11 @@ class Pip {
...
+// change 1
+//
+//
+//
+// change 3
+//
+// change 5
```
Comment (huaruiwu) [inline src/pip.js]: looks good

File: src/game.js [hunk 6]
```diff
@@ -187,3 +187,6 @@ module.exports.registerControls = function() {
...
 	})
 }
+
+//
```
Comment (huaruiwu) [inline src/game.js]: looks good

File: src/pip.js [hunk 7]
```diff
@@ -29,3 +29,11 @@ class Pip {
 }
 
 module.exports = Pip
+
+// change 1
+//
```
Comment (huaruiwu) [inline src/pip.js]: looks good

File: src/game.js [hunk 8]
```diff
@@ -187,3 +187,6 @@ module.exports.registerControls = function() {
...
+// test 2
```
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/game.js [hunk 9]
```diff
@@ -187,3 +187,5 @@ module.exports.registerControls = function() {
 		}
 	})
 }
+
+// test
```
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/game.js (hunk 8 above)
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/player.js [hunk 10]
```diff
@@ -109,3 +109,5 @@ class Player {
 }
 
 module.exports = Player
+
+//test
```
Comment (huaruiwu) [inline src/player.js]: exactly what I was going to do

File: src/player.js [hunk 11]
```diff
@@ -109,3 +109,5 @@ class Player {
 }
 
 module.exports = Player
+
```
Comment (huaruiwu) [inline src/player.js]: exactly what I was going to do

File: src/game.js [hunk 12]
```diff
@@ -187,3 +187,5 @@ module.exports.registerControls = function() {
 		}
 	})
 }
+
```
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/game.js (hunk 8 above)
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/game.js (hunk 6 above)
Comment (huaruiwu) [inline src/game.js]: exactly what I was going to do

File: src/pip.js (hunk 4 above)
Comment (huaruiwu) [inline src/pip.js]: this makes sense

File: src/game.js (hunk 8 above)
Comment (huaruiwu) [inline src/game.js]: this makes sense

File: src/pip.js (hunk 5 above)
Comment (huaruiwu) [inline src/pip.js]: exactly what I was going to do

File: src/pip.js [hunk 13]
```diff
@@ -29,3 +29,11 @@ class Pip {
 }
 
 module.exports = Pip
+
+// change 1
```
Comment (huaruiwu) [inline src/pip.js]: this makes sense

File: src/player.js (hunk 11 above)
Comment (huaruiwu) [inline src/player.js]: this makes sense

File: src/player.js (hunk 10 above)
Comment (huaruiw) [inline src/player.js:113]: exactly what I was going to do

File: src/player.js (hunk 11 above)
Comment (huaruiw) [inline src/player.js:112]: exactly what I was going to do

File: src/game.js (hunk 12 above)
Comment (huaruiw) [inline src/game.js:190]: exactly what I was going to do

File: src/game.js (hunk 9 above)
Comment (huaruiw) [inline src/game.js]: exactly what I was going to do

File: src/game.js (hunk 6 above)
Comment (huaruiw) [inline src/game.js:191]: exactly what I was going to do

File: src/game.js (hunk 8 above)
Comment (huaruiw) [inline src/game.js:192]: this makes sense

File: src/game.js (hunk 9 above)
Comment (huaruiw) [inline src/game.js]: this code belongs in a museum!

File: src/game.js (hunk 12 above)
Comment (huaruiw) [inline src/game.js:190]: exactly what I was going to do

File: src/player.js (hunk 11 above)
Comment (huaruiw) [inline src/player.js:112]: this code belongs in a museum!

File: src/player.js (hunk 10 above)
Comment (huaruiw) [inline src/player.js:113]: brought a tear to my eye

File: src/pip.js (hunk 3 above)
Comment (huaruiw) [inline src/pip.js:36]: I will definitely use this

File: src/pip.js [hunk 14]
```diff
@@ -29,3 +29,11 @@ class Pip {
...
+//
+//
```
Comment (huaruiw) [inline src/pip.js:35]: exactly what I was going to do

File: src/pip.js (hunk 7 above)
Comment (huaruiw) [inline src/pip.js:34]: this code belongs in a museum!

File: src/game.js (hunk 6 above)
Comment (huaruiw) [inline src/game.js:191]: brought a tear to my eye

File: src/game.js (hunk 8 above)
Comment (huaruiw) [inline src/game.js:192]: you are simply a genius!

File: src/game.js (hunk 6 above)
Comment (huaruiw) [inline src/game.js:191]: couldn't have written it better myself

File: src/game.js (hunk 8 above)
Comment (huaruiw) [inline src/game.js:192]: you are simply a genius!

File: src/game.js (hunk 12 above)
Comment (huaruiw) [inline src/game.js:190]: drivkin would be proud

File: src/game.js (hunk 9 above)
Comment (huaruiw) [inline src/game.js]: this code belongs in a museum!

File: src/player.js (hunk 11 above)
Comment (huaruiw) [inline src/player.js:112]: this code belongs in a museum!

File: src/player.js (hunk 10 above)
Comment (huaruiw) [inline src/player.js:113]: this code belongs in a museum!

**Pull request closed without merging**